docs/index.html

```diff
@@ -5,7 +5,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>Rivet Switch</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="https://assets.uits.iu.edu/css/rivet/1.0.0/rivet.min.css">
+  <link rel="stylesheet" href="https://assets.uits.iu.edu/css/rivet/1.1.0-alpha.2/rivet.min.css">
```

Comment:
if we aren't going to merge this until the 1.1.0 release then let's just make this point to `1.1.0`